parseorders.cpp

```diff
@@ -638,10 +638,8 @@ void Game::ProcessOrder(int orderNum, Unit *unit, AString *o,
 			ProcessWorkOrder(unit, 0, pCheck);
 			break;
 		case O_TRANSPORT:
-			ProcessTransportOrder(unit, o, pCheck);
-			break;
 		case O_DISTRIBUTE:
-			ProcessDistributeOrder(unit, o, pCheck);
+			ProcessTransportOrder(unit, o, pCheck);
```

Comment:
the `DISTRIBUTE` order will still be parsed (i.e., it's a valid alias for `TRANSPORT`).  There is no reason to have the commands be distinct really but the alias was left for legacy purposes.